e2e/Pages/TopBarComponent.ts

```diff
@@ -0,0 +1,22 @@
+import { expect } from '@playwright/test';
+import { Locator, Page } from 'playwright';
+
+import DashboardPage from './DashboardPage';
+
+class TopBarComponent extends DashboardPage {
```

Comment:
Let's use the naming convention as defined in the IBF-dashboard. For this test, it would be [disaster-type](https://github.com/rodekruis/IBF-system/tree/5bb38d45f41e84c4e890b04315f7593b8bdb8be7/interfaces/IBF-dashboard/src/app/components/disaster-type).

Reply:
DisasterTypeComponent